Detectors/TRD/calibration/macros/checkNoisyMCMs.C

```diff
@@ -106,11 +106,12 @@ void checkNoisyMCMs(std::string inpFile = "trdtracklets.root", bool useMean = tr
   auto n = nTrkltsPerActiveMcm.size() / 2;
   auto median = nTrkltsPerActiveMcm[n]; // the distinction between odd/even number of entries in the vector is irrelevant
   mean90 /= nActiveMcms90;
-  auto noiseThreshold = (useMean) ? 100.f * mean90 : (float)totalTriggerCounter / 10.f;
+  auto noiseThreshold = (useMean) ? 30.f * mean90 : (float)totalTriggerCounter / 5.f;
```

Comment:
Why are you tightening the definition of a noisyMCM, but at the same time loosening if useMean=false?
So in the first case, a noisyMCM is now one which produced 30x the mean of the lower 90 perecent.
In the second case, a noisyMCM is one which triggered on twice as much as before.

Reply:
Am I not tightening both? if useMean==false then a MCM which fired for at least 20% of the triggers is noisy, before it was only in case it fired 10% of the time. And I did that, because with the previous settings it looked to me we were not strict enough (looking at how many tracklets were sent by MCMs which were previously not marked noisy in comparison to the ones marked as noisy). I am trying to find the spot were there is a sudden jump for the number of tracklets sent per MCM

Reply:
I looked a bit further into this and now I am not entirely convinced this is as effective as we think (given I do not fully understand what _really_ is a `noisy` MCM.

Below I plotted the _sorted_ vector with trackletCounter[i].first as bin content e.g. the number of tracklets. (This is to say that the shown xaxis is not the actual mcm number but in fact the idx after sort, hence should be disregarded for my argument, also the xaxis is zoomed since most of the mcm produce roughly an even number of tracklets, i used 10000tfs)
Additionally I drew the value for noiseThreshold as a red line.

There seems to be this small tail under red line we cannot really get rid of using these methods.
Like 200 MCMs.
Is this acceptable or should we tighten this even more?
Also below is the same for another Run with different triggers, there it looks like roughly 500 MCM in this tail.

526055,useMean=true:
![image](https://user-images.githubusercontent.com/42321018/193769615-560adb4b-1cb8-49c9-86af-4d6f106c3eb7.png)

526055,useMean=false:
![image](https://user-images.githubusercontent.com/42321018/193769770-4b370bc7-22cb-4d56-afb3-da9cca295e91.png)

Reference Run 523677, depends on triggers:
![image](https://user-images.githubusercontent.com/42321018/193769898-8a5b214c-70b7-4f0c-a530-d4267620a8e0.png)

---
This is how these plots look not zoomed in, most of the stuff happens at the end:
![image](https://user-images.githubusercontent.com/42321018/193771388-e20a88e5-df6c-4527-89ef-651585a68f3f.png)

Reply:
Hi Felix,
in the end its not really MCMs which are noisy, at least as far as I can tell, but individual pads instead. See for example my comment here: https://github.com/AliceO2Group/AliceO2/pull/7615#issuecomment-970052729
Once we are able to take proper noise runs we should be able to detect noisy pads automatically and mask them. Afterwards no noisy MCM should remain.
The noise threshold we have set on the level as we are doing now is always kind of arbitrary. But this should be OK, since its used only for QC to clear the spectra so that we are able to see other peculiarities of the data. Without it e.g. the plots for charges would be polluted by the tracklets from only few MCMs.
I don't think it is worth to put much effort into optimizing the noisy MCM finder further. Instead, we should focus on the pad level analysis which @Archita-Dash is looking into